installer/volcano-development.yaml

```diff
@@ -5227,7 +5230,7 @@ webhooks:
         namespace: volcano-system
         path: /pods/validate
         port: 443
-    failurePolicy: Fail
+    failurePolicy: Ignore
```

Comment:
same as above

Reply:
Let me explain my point of view, why failurePolicy should is Fail in this place. 

When a pod scheduler name is `volcano`, but volcano webhook server not running, this time kube-apiserver call volcano webhook will fail, the pod will be paused here.

If we change failurePolicy from Fail to Ignore, then kube-apiserver can skip volcano webhook to volcano scheduler step, without MutatingWebhookConfiguration, can the scheduling achieve the expected results?

Reply:
Yes, So should we keep  failurePolicy is Fail?

I add `expression: object.spec.schedulerName == '{{- $scheduler_name }}'` this condition is because, i hope when volcano wenhook server not running whatevery the reason, don't influence not use volcano scheduler pod.

Reply:
volcano webhook is a important inspection component, It is not an insignificant auxiliary component.
So if volcano webhook is down. All workloads that need to be scheduled using volcano should be blocked.
We should keep failurePolicy is Fail.

I agree with your other idea, if the workload is not scheduled using volcano, it should not be affected when the volcano webhook fails.

Reply:
thanks ~ 🤝

Reply:
When we didn't add match condition of scheduler name, if the pod is not scheduled by volcano, the  policy can be `Ignore` to not block pods scheduled by default scheduler, if we have exactly added the match condition and be sured that pod is scheduled by volcano, then it's reasonable to keep policy `Fail`.

Reply:
This has not been resolved